src/main/resources/templates/fragments/topNav.html

```diff
@@ -1,3 +1,4 @@
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
```

Comment:
imported thymeleaf to resolve issue

Reply:
Since this is a fragment, thymeleaf is likely included in the master layout.  No need to add it again here.